Products/CMFQuickInstallerTool/interfaces/portal_quickinstaller.py

```diff
@@ -22,7 +22,10 @@ def isProductInstallable(productname):
 
     def isProductAvailable(productname):
         ''' is the product directory present (to check if it has been deleted
-            from the Filesystem '''
+            from the Filesystem
+
+        Deprecated: use isProductInstallable instead.
```

Comment:
Should we raise a warning here? https://docs.plone.org/develop/styleguide/deprecation.html

Reply:
This is only in an interface definition. The implementation of this method in the tool itself already gives a proper warning (even in Plone 4).